src/library/fixbook/FixBookUI.java

```diff
@@ -1,26 +1,37 @@
 package library.fixbook;
 import java.util.Scanner;
 
+// Author : Sankalpa
+// Reviewer : Chathura
+// Mediator : Poornima	
 
 public class FixBookUI {
 
-	public static enum uI_sTaTe { INITIALISED, READY, FIXING, COMPLETED };
+	//public static enum uI_sTaTe { INITIALISED, READY, FIXING, COMPLETED };
+	public static enum uiState { INITIALISED, READY, FIXING, COMPLETED }; // changed enum name uI_sTaTe to uiState
 
-	private fIX_bOOK_cONTROL CoNtRoL;
-	private Scanner InPuT;
-	private uI_sTaTe StAtE;
+	//private fIX_bOOK_cONTROL CoNtRoL;
+	private FixBookControl control; // changed variable type  fIX_bOOK_cONTROL to FixBookControl and CoNtRoL to control
+	//private Scanner InPuT;
+	private Scanner input; // changed variable name InPuT to input
+	//private uI_sTaTe StAtE;
+	private uiState state; // changed enum name uI_sTaTe to uiState
 
 	
-	public FixBookUI(fIX_bOOK_cONTROL CoNtRoL) {
-		this.CoNtRoL = CoNtRoL;
-		InPuT = new Scanner(System.in);
-		StAtE = uI_sTaTe.INITIALISED;
-		CoNtRoL.SeT_Ui(this);
+	public FixBookUI(FixBookControl control) {
+		//this.CoNtRoL = CoNtRoL;
+		this.control = control; //changed variable CoNtRoL to control
+		//InPuT = new Scanner(System.in);
+		input = new Scanner(System.in);
+		//StAtE = uI_sTaTe.INITIALISED;
+		state = uiState.INITIALISED; // changed enum name uI_sTaTe to uiState

```

Comment:
enum names should be treated in a similar manner as class names when modifying them to proper format. Thank you.